data/hails.txt

```diff
@@ -700,9 +747,12 @@ phrase "friendly civilian"
 		"rotten fruit"
 		"sludge"
 		"unprocessed sewage"
+		"nuclear waste"
+		#add more smelly stuff
```

Comment:
Unneeded comment

Reply:
And nuclear waste doesn't smell. It just hurts your cells. Honestly, I think this entire hail is fine as is. No need to add more.